src/Deserializers/FacetContainerDeserializer.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Wikibase\DataModel\Serializers;
+
+use Deserializers\DispatchableDeserializer;
+use Wikibase\DataModel\Facet\FacetContainer;
+use Wikimedia\Assert\Assert;
+
+/**
+ * Package private
+ *
+ * @since 2.1
+ *
+ * @licence GNU GPL v2+
+ * @author Daniel Kinzler
+ */
+class FacetContainerDeserializer  {
```

Comment:
This name confused me, as this actually does not implement the deserializer interface